### DISCUSSION TITLE
Grenade launcher damage

### DISCUSSION THREAD

**Discussion by mithrand0** [ReactiveDrop/reactivedrop_public_src#301] [General]
> Ax-Now Sunday 10:44:
> Guys the devs changed the gl on the beta so that the grenade inflicts x2 damage instead of twice damage (as is the case now) which created a problem with the harvester because, in brutal, twice damage killed the xeno in 3nades with jaeger crash and sarge in solo, x2 damage left the xenos alive with 3nades on crash and sarge, instead of leaving the gl as it is they changed the harvester which no longer produces xenos when it dies of gl, so jaeger kills it in 2 nades and all other characters kill it in 3 nades. This didn't solve the problem as I expected. point out this problem at the base I wanted him to realize when gl changes like that he will change the balance between the characters, with the change on harvester it has only moved the problem, all charcaters are now able to kill the harvester in 3 nades direct hit, and the alien has lost a lot of interest because it no longer spawns xenos when it dies of gl
> and I'm not the only player thought it's a bad change
there is no speech from the developers, I do not know if they realized that it was a bad idea or if they are simply in tunnel vision on their work or if he refuses to listen to the opinions of players experienced by simple ego
> I try by this message to wake up the community so that it is a little more active and watchful on what is happening on the game. for that you all have access to the beta of the game in the properties of the steam game and the forums / discord to express yourself. Thanks

> jhheight [Sunday 12:58]
> 	i would say its hard to predict, but it is not random. for example on brutal 3 players harvester has 480 health. jaeger's nades do 120 damage per explosion and since nades explode twice when hitting directly that means ideally 240 damage. however directly hitting the harvester the grenade does from 97% to 98% the damage from explosion, meaning after 2 nades the harvester will be at 20 hp or so. so for 3 players the strategy is to shoot the harvester once or twice with the vindicator and then after 2 nades it dies without releasing the xenomytes

Image: https://steamuserimages-a.akamaihd.net/ugc/1815536420213953144/B23CFF210D2446DE36BB37F63FCD329767691AC5/

> 	what i wanna say is that theres couple factors which is the explosion bonus you have and the difficulty with amount of players,  after a lot of playing you could learn to get a memory for those factors to kill harvesters more efficiently, there can be cases where slightly damaging the harvester before nading it will make it not release xenos

> [Sunday 13:45] trick-master:
>	whether it's from nade or saw or whatnot, not having the xeno burst out feels weird though.yes, it was/is nonsense when you saw and insta die from xeno, same with nade, but imo that's part of the fun.not a fan of making everything easier for the sake of consistency (*cough* buzzer blur *cough*) :p

> [Sunday 14:53] SLH: You don't get hit by xenomites if you know how to position when you saw the harvester. Also, why spend time messing with GL or other useful weapons/items when balancing useless items and weapons would be time better spent? I'm pretty sure most people would prefer having more than 5 useful items over changing and most likely messing up GL somehow. Having useful hornet barrage, smartbomb, tesla coils, PDWs, cannon sentry, etc would make the game a lot more fun. The buff to sniper and beacons were great and made the game a lot more enjoyable. Although I can't say the same about the gas change considering the circle/marker is broken and the animation is too much. My point is...don't break the things that are working well for the most part. It's better to try to balance the useless stuff.

> [Sunday 23:09] Ben Lubar:
> 	I'm only changing grenade launcher because I want it to be consistent and not rely on a bug to get its damage. The goal is to have it be pretty much the same as it was before, just without weird double hit interactions.
>	There's definitely space to improve some of the less-used weapons, but that's going to come after I finish the swarmopedia weapon pages unless someone else does it.

> [Sunday 23:37] jhheight:
>	to make it same as before it must be that harvesters will release xenomytes when their health was half or less of the grenade damage

> [Yesterday, 08:14] Ax-now:
> 	 its not inconsistant and its not a bug
>	i miss Dmitriy, he was careful not break the game. you take too many decisions without thinking and do not listen to the advice of good players who know the game, i have the impression that alien swarm is just a excuse for a good playground to code for you. :(

> I regret showing you the problem on the harvester, if I hadn't done anything, only the gl would have had modifications and you would never have realized that the harv would need more grenades with crash sarge because you don't never test anything after finishing your code and the harvester would never have undergone unjustified change

**Comment by jhh8** [comment]
currently in beta branch the conditions for harvester not releasing xenomytes are these:
- rd_harvester_always_release_xenomites convar must be 0 (by default its 0)
- damage type with which harvester died must be an explosion
- damage amount must be over 135
- damage amount must be at least 2 times higher than harvester's health before the damage was dealt

**Comment by mithrand0** [comment]
Steam discussion related to this https://steamcommunity.com/app/563560/discussions/2/3361397532251132492/

**Comment by mithrand0** [comment]
This topic is quite old, and no balance changes are planned for the GL right now. Closing it.
